plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -763,20 +763,6 @@ public static function create_pages() {
 
 		include_once dirname( __FILE__ ) . '/admin/wc-admin-functions.php';
 
-		/**
-		 * Determines the cart shortcode tag used for the cart page.
-		 *
-		 * @since 2.1.0
-		 */
-		$cart_shortcode = apply_filters( 'woocommerce_cart_shortcode_tag', 'woocommerce_cart' );
-
-		/**
-		 * Determines the checkout shortcode tag used on the checkout page.
-		 *
-		 * @since 2.1.0
-		 */
-		$checkout_shortcode = apply_filters( 'woocommerce_checkout_shortcode_tag', 'woocommerce_checkout' );
-
```

Comment:
Maybe consider using `apply_filters_deprecated` , if it helps

https://developer.wordpress.org/reference/functions/apply_filters_deprecated/

Reply:
@PanosSynetos is this something that should be used even if are not replacing it with a new filter or using the output at all?

Should I use `apply_filters_deprecated` with a false default value, and use that output as page content if !== false?
this would keep support to any theme/plugin implementing a custom cart/checkout via custom shortcode tags?

@senadir @mikejolley thoughts?

Reply:
This only applied to default content which no longer uses shortcodes. The filters run at installation time also; I wonder what use cases needed this if any?

`woocommerce_create_pages` filter can be used to change the content too FYI.

Reply:
It was just a suggestion, so that if anyone is using those filters, they would get a log in their notice.

I don't really see a use case where these filters are needed, and I'm happy with a dev note.

On Woo Express, we're using the `woocommerce_create_pages` to set blocks as default

Reply:
Good to hear from an actual host Panos! I think the use case for them us hosting providers who might insert a different shortcode, in our case, no shortcodes are used anymore, so hosts can follow your same route and use `woocommerce_create_pages`

Reply:
Added a dev note, and chose to deprecate the filters safely.
https://github.com/woocommerce/woocommerce/pull/40867/commits/5a247444d4da2c8e10165738c8092fd7536b598b